README.md

```diff
@@ -9,13 +9,13 @@ This enables the users of SDKMAN to contribute new Installation Candidates and r
 
 This repo uses [mongobee](https://github.com/mongobee/mongobee) as database migration framework. In order to contribute a PR, it is required to have a local installation of MongoDB on your machine. Alternatively run it up with Docker (works on Linux):
 
-        $ docker run -d --network=host --name=mongo mongo:3.2
+        $ docker run -d --network=host --name=mongo mongo:4.0
```

Comment:
The reason that we want to run it in 3.2 is to ensure that migrations are compatible with what we have in our prod environment. Mind if we leave this as it was?

Reply:
NP